iana.md

```diff
@@ -4,6 +4,26 @@
 
 ## ChapterCodecID Registry
 
+## MIME Types
+
+Matroska files and streams are found in three main forms: audio-video files, audio-only and occasionally with stereoscopic video tracks.
+
+The MIME types to use for each type is:
+
+* "video/matroska" for streams containing video tracks
+* "audio/matroska" for streams containing audio tracks with no video tracks
+* "video/matroska-3d" for streams containing some stereoscopic video tracks
```

Comment:
Is `some` necessary? (Also below.)

Reply:
`any` would work too. It also works with and without `some`. I have no preference.

If we want stricter wording we should say that the MIME types `SHOULD` or `SHOULD NOT` be used if there are(not) audio/stereoscopic tracks.

Then there's the case where the file has no audio or video (pure chapter files, as is possible with DVD "images"). It should be `video/matroska` but then it doesn't contain any video.